### PR TITLE
Make samples run in parallel

### PR DESCRIPTION
**Pull request by alec-deason**

### CODE REVIEW

File: src/cascade/core/subprocess_utils.py [hunk 1]
```diff
@@ -46,23 +47,27 @@ def inner(text):
...
-        await std_out_task
-        await std_err_task
+    return sub_process.returncode, "".join(stdout_accumulator), "".join(stderr_accumulator)
 
-        return sub_process.returncode, "".join(stdout_accumulator), "".join(stderr_accumulator)
 
-    return loop.run_until_complete(coroutine())
+def run_with_logging(command):
+    loop = asyncio.get_event_loop()
+    if loop.is_running():
```
[the remark this answers is not in the document]
Comment (alec-deason) [reply src/cascade/core/subprocess_utils.py]: Exactly

File: src/cascade/executor/estimate_location.py [hunk 2]
```diff
@@ -277,18 +284,20 @@ def _fit_and_predict_fixed_effect_sample(sim_model, sim_data, fit_file, location
...
+                base_path / fit_file,
                 input_data.locations_df,
                 model.location_id,
                 local_settings,
                 draw_idx
             ))
 
         results = await asyncio.gather(*jobs)
-    return results
+    return [r for r in results if r is not None]
```
Comment (adolgert) [inline src/cascade/executor/estimate_location.py]: I wouldn't have guessed that.

File: src/cascade/executor/estimate_location.py [hunk 3]
```diff
@@ -277,18 +284,20 @@ def _fit_and_predict_fixed_effect_sample(sim_model, sim_data, fit_file, location
...
 
 
-def make_draws(model, input_data, max_fit, local_settings, num_processes):
+def make_draws(execution_context, model, input_data, max_fit, local_settings, num_processes):
+    base_path = execution_context.db_path(local_settings.parent_location_id)
+    base_path.mkdir(parents=True, exist_ok=True)
```
Comment (adolgert) [inline src/cascade/executor/estimate_location.py]: Nice job pulling that in.

File: src/cascade/executor/estimate_location.py (hunk 2 above)
Comment (alec-deason) [reply src/cascade/executor/estimate_location.py]: Hmm? I have to return something from the jobs, even when the fail, so here I filter the failures out.

**Pull request merged**